### DISCUSSION TITLE
Support for BLOB

### DISCUSSION THREAD

**Discussion by ecxs** [Budibase/budibase#8836] [Help] [answered]
Hi there,

Are there any plans to support BLOB type for external (MySQL) databases?

Thanks in advance, and best regards, Dick

**Comment by andz-bb** [comment, marked as answer]
Hi @ecxs,

At the minute there's no support for BLOB type or any other file attachment type for external databases in Budibase. There's an open issue to include support for them here: #4302 - if you want to see this added in the future you can add a +1 to the issue to show your support!

**Comment by ecxs** [comment]
Hi @andz-bb Where do I ad the thumbs up?

**Reply by andz-bb** [reply]
hey Dick, you can post a comment at the bottom of this thread to say it would be useful for you: https://github.com/Budibase/budibase/issues/4302
